guides/v2.2/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -77,8 +78,8 @@ Use this table to understand what changes Magento can make and which version num
 | | New optional method argument | MINOR|
 | | Removed a non-last argument| MAJOR|
 | | New required constructor object argument | MINOR|
-| | New optional constructor argument for [extendable @api][extendable-api]{:target="_blank"} | MINOR|
-| | New optional constructor argument | PATCH|
+| | New optional constructor argument in a class intended for extension | MINOR|
```

Comment:
Please give reference to the section listing these classes.

Reply:
Added